frontend/src/data/tasks.json

```diff
@@ -644,7 +644,7 @@
       {
         "answerKey": "married-or-cl",
         "descriptionEn": "If you want to split your eligible pension income with your spouse or partner, you will both need to complete and submit a joint election to split private pension income when you file your taxes. **This does not apply to your OAS and CPP pensions**.\n\nYou will need to gather the following information in order to apply for pension splitting:\n- The transferring spouse's or partner's T4A slips\n- The transferring spouse's or partner's T4A-RCA slips (if applicable)\n- The transferring spouse's or partner's veterans benefits amounts (if applicable)\n\nYou may want to talk with a financial professional about the impact of pension splitting before applying.",
-        "descriptionFr": "Si vous souhaitez fractionner votre revenu de pension admissible avec votre conjoint ou partenaire, vous devrez tous deux remplir et soumettre un choix conjoint de fractionnement du revenu de pension privée au moment de remplir votre déclaration de revenus. **Cela ne s'applique pas à vos pensions de la SV et du RPC**.\n\nVous devrez réunir les renseignements suivants pour demander le fractionnement du revenu de pension\u00a0:\n- les feuillets T4A du conjoint ou partenaire qui effectue le transfert\n- Les feuillets T4A-RCA du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n- Montant des prestations d'ancien combattant du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n\nVous pouvez discuter avec un expert financier des conséquences du fractionnement du revenu de pension avant de déposer votre demande.",
+        "descriptionFr": "Si vous souhaitez fractionner votre revenu de pension admissible avec votre conjoint ou partenaire, vous devrez tous deux remplir et soumettre un choix conjoint de fractionnement du revenu de pension privée au moment de remplir votre déclaration de revenus. **Cela ne s'applique pas à vos pensions de la SV et du RPC**.\n\nVous devrez réunir les renseignements suivants pour demander le fractionnement du revenu de pension\u00a0:\n- les feuillets\u00a0T4A du conjoint ou partenaire qui effectue le transfert\n- Les feuillets\u00a0T4A-RCA du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n- Montant des prestations d'ancien combattant du conjoint ou partenaire qui effectue le transfert (le cas échéant)\n\nVous pouvez discuter avec un expert financier des conséquences du fractionnement du revenu de pension avant de déposer votre demande.",
```

Comment:
Non-breaking space `\u00a0` is not needed with `feuillets T4A` but change `-` for a non-breaking hypen `\u2011` with `T4A-RCA`.

Reply:
removed `\u00a0` before feuillets T4A and add T4A`\u2011`RCA